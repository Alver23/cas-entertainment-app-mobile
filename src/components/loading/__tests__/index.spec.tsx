// Dependencies
import React from 'react';
import { shallow, ShallowWrapper } from 'enzyme';

import Loading from '../index';

describe('<Loading />', () => {
  let component: ShallowWrapper;

  const props = {};

  beforeEach(() => {
    component = shallow(<Loading {...props} />);
  });

  afterEach(() => {
    jest.clearAllMocks();
  });

  it('should save a snapshot of the component', () => {
    expect(component).toMatchSnapshot();
  });
});
