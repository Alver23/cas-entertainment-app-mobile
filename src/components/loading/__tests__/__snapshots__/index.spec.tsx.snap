// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loading /> should save a snapshot of the component 1`] = `
<ActivityIndicator
  animating={true}
  color="#2089dc"
  hidesWhenStopped={true}
  size="large"
/>
`;
